gempython/tools/optohybrid_user_functions_xhal.py

```diff
@@ -163,7 +163,7 @@ def checkSbitMappingWithCalPulse(self, **kwargs):
         L1AInterval - Number of BX's inbetween L1A's
         mask        - VFAT mask to use for excluding vfats from the trigger
         nevts       - Number of events for each dac value in scan
-        outData     - Pointer to an array of size (24*128*8*nevts) which
+        outData     - Pointer to an array of size (self.parentAMC.nVFAT*128*8*nevts) which
```

Comment:
small issue, `parentAMC` does not have a data member (nor should it) called `nVFAT`.  I think there you mean `vfatsPerGemVariant[self.gemType]`?